src/main/resources/templates/index.html

```diff
@@ -1,22 +1,32 @@
-<!DOCTYPE html>
-<html lang="en" xmlns="http://www.w3.org/1999/xhtml"
-      xmlns:th="http://www.thymeleaf.org">
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org">
 <head>
+    <title>Jstat Software</title>
+
+    <div th:replace="fragments/header :: header-css"/>
 
-<head>
-    <meta charset="UTF-8">
-    <title>JState-Gui</title>
 </head>
 <body>
-<!-- This is the header-->
-<h1>Welcome to JStat</h1>
-<img th:src="@{/images/TacoCloud.png}"/>
-
-<div>
-    <ul>
-        <li><a th:href="@{/quit}">Quit</a></li>
-        <li><a th:href="@{/load-dataset}">Load Dataset</a></li>
-    </ul>
+
+<div th:replace="fragments/header :: header"/>
+
+<div class="container">
+
+    <div class="starter-template">
+        <h1>Welcome to Jstat</h1>
+        <p>What would you like to do today?</p>
+    </div>
+    <div>
+        <ul>
+            <li><a th:href="@{/load-dataset}">Load Dataset</a></li>
```

Comment:
I think these should go in the header...the header manages the Navigation right? If yes then we should move them to the ```header.html```